WordPress/Classes/ViewRelated/Notifications/Tools/NotificationReplyStore.swift

```diff
@@ -0,0 +1,133 @@
+import Foundation
+
+
+// MARK: - NotificationReplyStore
+//
+class NotificationReplyStore {
+
+    /// Shared Instance.
+    ///
+    static let shared = NotificationReplyStore()
+
+    /// Unit Testing Helper: Allows us to hack the current date.
+    ///
+    private var overridenNow: Date?
+
+    /// Our beautiful and private Initializer. In here we'll trigger the cleanup sequence, which removes outdated replies!.
+    ///
+    private init() {
+        purgeOldReplies()
+    }
+
+    /// Unit Testing Helper: Allows us to hack the current date.
+    ///
+    init(now: Date) {
```

Comment:
You probably can mark this private also? and then in the tests say that you want access to the private methods.

Reply:
I would absolutely love to do so. But the `@testable import` trick, in Unit Tests, only lets you call *internal* methods, not private.

I really really hope Swift 5 comes with annotations!!! thanks for the suggestion though!!